lnbits/wallets/lnbits.py

```diff
@@ -12,11 +13,16 @@ def __init__(self):
         self.auth_admin = {"X-Api-Key": getenv("LNBITS_ADMIN_KEY")}
         self.auth_invoice = {"X-Api-Key": getenv("LNBITS_INVOICE_KEY")}
 
-    def create_invoice(self, amount: int, memo: str = "") -> InvoiceResponse:
+    def create_invoice(self, amount: int, memo: str = "", description_hash: bytes = b"") -> InvoiceResponse:
         r = post(
             url=f"{self.endpoint}/api/v1/payments",
             headers=self.auth_invoice,
-            json={"out": False, "amount": amount, "memo": memo}
+            json={
+                "out": False,
+                "amount": amount,
+                "memo": memo,
+                "description_hash": hexlify(description_hash).decode("ascii"),
```

Comment:
`bytes.hex()` can be used here directly => `description_hash.hex()`
https://docs.python.org/3/library/stdtypes.html#bytes.hex